spec/models/edition_spec.rb

```diff
@@ -234,15 +234,15 @@
     end
   end
 
-  describe "#change_note_html" do
+  describe "#reason_for_change_html" do
```

Comment:
I can't see where this is getting used..

It only occured to me to search for it as we're applying simple_format to the reason for change in the frontend and it didn't make sense to have html tags in there.